components/calendar/src/astronomy.rs

```diff
@@ -771,8 +771,8 @@ impl Astronomical {
     /// Based on functions from _Calendrical Calculations_ by Reingold & Dershowitz,
```

Comment:
nit: this function no longer takes a moment, docs should change

Reply:
fixed.